src/components/Search/SearchResults/SearchResults.tsx

```diff
@@ -40,11 +41,13 @@ const SearchResults = ({
       <ul data-testid="search-results" className={`${px}-search__results-container`}>
         {isLoading ? <li className={`${px}-search__result`}>{loadingText}</li> : null}
         {hasResults &&
-          autoCompleteResults.map((result) => {
+          autoCompleteResults.map((result, i) => {
             return (
-              <li key={result.id} className={`${px}-search__result`}>
+              <li key={result.id} className={`${px}-search__result`} onClick={() => showSearchResults(false)}>
```

Comment:
might be good to put this onClick on the `Link` component? what you think @scottdickerson